examples/training_examples/train_example.py

```
@@ -0,0 +1,109 @@
+    print(f"JK mode: {config['model']['jk_mode']}")
+
+    # Initialize trainer with config
+    trainer = create_trainer(config=config, train_dir=args.train_dir, val_dir=args.val_dir)
+
+    # Define loss functions (MSE for regression tasks)
+    node_loss_fn = nn.MSELoss()
+    graph_loss_fn = nn.MSELoss()
+
+    # Train model
+    print(f"Starting training for {config['training']['epochs']} epochs...")
+    history = trainer.train(node_loss_fn=node_loss_fn, graph_loss_fn=graph_loss_fn, checkpoint_dir=args.output_dir)
```

Ruff (0.11.9)

Local variable `history` is assigned to but never used

Remove assignment to unused variable `history`

(F841)